sdk/dotnet/Yaml/Yaml.cs

```diff
@@ -1497,174 +1067,84 @@ internal static class Parser
                         };
-                    case "storage.k8s.io/v1/StorageClassList":
-                        return new[]
-                        {
-                            id.Apply(id => ($"storage.k8s.io/v1/StorageClassList::{id}",
-                                new Storage.V1.StorageClassList(id, obj!, opts) as KubernetesResource))
-                        };
                     case "storage.k8s.io/v1/VolumeAttachment":
                         return new[]
                         {
                             id.Apply(id => ($"storage.k8s.io/v1/VolumeAttachment::{id}",
                                 new Storage.V1.VolumeAttachment(id, obj!, opts) as KubernetesResource))
                         };
-                    case "storage.k8s.io/v1/VolumeAttachmentList":
-                        return new[]
-                        {
-                            id.Apply(id => ($"storage.k8s.io/v1/VolumeAttachmentList::{id}",
-                                new Storage.V1.VolumeAttachmentList(id, obj!, opts) as KubernetesResource))
-                        };
                     case "storage.k8s.io/v1alpha1/VolumeAttachment":
                         return new[]
                         {
                             id.Apply(id => ($"storage.k8s.io/v1alpha1/VolumeAttachment::{id}",
                                 new Storage.V1Alpha1.VolumeAttachment(id, obj!, opts) as KubernetesResource))
                         };
-                    case "storage.k8s.io/v1alpha1/VolumeAttachmentList":
-                        return new[]
-                        {
-                            id.Apply(id => ($"storage.k8s.io/v1alpha1/VolumeAttachmentList::{id}",
-                                new Storage.V1Alpha1.VolumeAttachmentList(id, obj!, opts) as KubernetesResource))
-                        };
                     case "storage.k8s.io/v1beta1/CSIDriver":
                         return new[]
                         {
                             id.Apply(id => ($"storage.k8s.io/v1beta1/CSIDriver::{id}",
                                 new Storage.V1Beta1.CSIDriver(id, obj!, opts) as KubernetesResource))
                         };
-                    case "storage.k8s.io/v1beta1/CSIDriverList":
-                        return new[]
-                        {
-                            id.Apply(id => ($"storage.k8s.io/v1beta1/CSIDriverList::{id}",
-                                new Storage.V1Beta1.CSIDriverList(id, obj!, opts) as KubernetesResource))
-                        };
                     case "storage.k8s.io/v1beta1/CSINode":
                         return new[]
                         {
                             id.Apply(id => ($"storage.k8s.io/v1beta1/CSINode::{id}",
                                 new Storage.V1Beta1.CSINode(id, obj!, opts) as KubernetesResource))
                         };
-                    case "storage.k8s.io/v1beta1/CSINodeList":
```

Comment:
Why are all these missing in the new version? Looks like all the `List` kinds.

Reply:
Yeah, all of the list cases were redundant (not used). The preceding conditional https://github.com/pulumi/pulumi-kubernetes/pull/1184/files?file-filters%5B%5D=.cs#diff-4a352fae8d28400c9f379226c4ec9c1bR419-R508 handles all of the list kinds.